### DISCUSSION TITLE
I found a Kinect 360 in my parts bin!   I know it worked with OpenCV 4 years ago,  can I use it with ROS2 ? Or am I better off with an Oak D Lite due to the lower 2.5w power requirement ?

### DISCUSSION THREAD

**Discussion by jerrynsr** [iRobotEducation/create3_docs#53] [Ideas]
I was in my basement looking through my parts bins for hardware I can use with the new Create3 bot.  Basically I'm  building a Turtlebot4 out of it and using some of my own mods.   What's the status of Kinnect 360 support with the latest and greatest version of OpenCV and ROS2 ?    The power requirement  for the Kinect 360 is 1 amp.  I can shorten the sub data cable, remove the connector for the power cable and connect the 12v power cable provided with the create 3 to it.       on the other hand the Oak D lite consumest much less power,   minus the microphone array but robot runtime will be longer.  I have not found a source in the USA with stock of a Oak D lite fixed focus version.  Sparkfun just has stock of the autofocus version and does not have the split usb power cable.  If I decide on the Oak D lite I plan on connecting a 12v->USBC power adapter to the 12v power source on the create 3 interface.

**Comment by shamlian** [comment]
This question might be better asked at [ROS Discourse](https://discourse.ros.org/).
